services/brig/src/Brig/API/User.hs

```diff
@@ -155,7 +155,7 @@ createUser new@NewUser{..} = do
     -- Create account
     (account, pw) <- lift $ newAccount new { newUserIdentity = ident } (Team.inInvitation . fst <$> teamInvitation) tid
     let uid = userId (accountUser account)
-
+    Log.debug $ field "user" (toByteString uid) . msg (val "User.createUser")
     Log.info $ field "user" (toByteString uid) . msg (val "Creating user")
```

Comment:
Can we just move `User.createUser` to an `action` field?

Reply:
Do you want to do that for the other ones as well? From the previous PR

Reply:
Yes, would be great 🙇